collections/_docs/en/001-001-install.md

```diff
@@ -16,7 +16,7 @@ For the installation process watch the video for your operating system
 
 ## Requirements
 
-* [Inkscape](https://inkscape.org/) Version 0.92.2 or higher
+* [Inkscape](https://inkscape.org/) Version 0.92.2 and [Inkscape](https://inkscape.org/release/inkscape-1.0/?latest=1) Version 1.0 Release Candidate.
```

Comment:
Maybe we should say Inkscape Version 0.92.2 or higher (including Inkscape Version 1.0 [...])

Current version is 0.92.5, we don't want to drop support on this one ;)

Reply:
Ahh, yes, or else it implies it doesn't work with 0.92+
Fixed.